client/src/pages/SplashScreen/SplashScreen.tsx

```diff
@@ -0,0 +1,78 @@
+import { Grid, TextField, Button, Typography, CircularProgress } from '@material-ui/core';
+import * as Yup from 'yup';
+import { useFormik } from 'formik';
+import useStyles from './splashScreenStyles';
+import { useSnackBar, useKanban } from '../../context';
+
+interface Values {
+  name: string;
+}
+
+const SplashScreen = (): JSX.Element => {
+  const classes = useStyles();
+  const { createNewBoard, sendToFirstBoard } = useKanban();
+  const { updateSnackBarMessage } = useSnackBar();
+
+  const handleSubmit = async ({ name }: Values): Promise<void> => {
+    const { error } = await createNewBoard(name);
+
+    if (error) {
+      updateSnackBarMessage(error, 'error');
+
+      return;
+    }
+    updateSnackBarMessage('Board created!');
+    sendToFirstBoard();
+  };
+
+  const formik = useFormik({
+    initialValues: {
+      name: '',
+    },
+    validationSchema: Yup.object({
+      name: Yup.string()
+        .min(3, 'Minimum of 3 characters length!')
+        .max(50, 'Maximum of 50 characters length!')
+        .required('Required field!'),
+    }),
+    onSubmit: (values: Values) => handleSubmit(values),
+  });
+  return (
+    <Grid xs={12} container direction="column" justify="center" spacing={3} className={classes.root}>
```

Comment:
Grid containers don't have support for `xs={12}`

Reply:
Sure, thank you!

Reply:
I did this all over the parts of the UI that I wrote before I noticed the errors showing up in the console. If you want to apply a responsive grid size to the container itself you'd have to wrap it in another `<Grid item />`

Reply:
Thanks, Jon!